src/components/sidebar/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { styled } from '@mui/material/styles';
+import {
+  Box,
+  Drawer,
+  CssBaseline,
+  List,
+  Divider,
+  IconButton,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from '@mui/material';
+import { Menu, ChevronLeft } from '@mui/icons-material';
+import {
+  GridOnOutlined,
+  DescriptionOutlined,
+  PlagiarismOutlined,
+  LaptopWindowsOutlined,
+  ScreenSearchDesktopOutlined,
+  SettingsOutlined,
+  PersonOutlineOutlined,
+} from '@mui/icons-material';
+
+const drawerWidth = 240;
+
+const DrawerHeader = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  padding: theme.spacing(0, 1),
+  ...theme.mixins.toolbar,
+  justifyContent: 'flex-end',
+}));
+
+const sidebarButtonsList = [
+  {
+    key: 'Monitor',
+    Icon: GridOnOutlined,
+  },
+  {
+    key: 'Document',
+    Icon: DescriptionOutlined,
+  },
+  {
+    key: 'Cautare Documente',
+    Icon: PlagiarismOutlined,
+  },
+  {
+    key: 'Proiect',
+    Icon: LaptopWindowsOutlined,
+  },
+  {
+    key: 'Cautare Proiecte',
+    Icon: ScreenSearchDesktopOutlined,
+  },
+  {
+    key: 'Optiuni',
+    Icon: SettingsOutlined,
+  },
+  {
+    key: 'Profilul meu',
+    Icon: PersonOutlineOutlined,
+  },
+];
+
+export default function Sidebar({ sidebarState }: any) {
+  const [open, setOpen] = useState(true);
+
+  const handleDrawerOpen = () => {
```

Comment:
These functions should be extracted from the global context:
`const {closeDrawer, openDrawer} = useContext(InteractiveComponentsContext)` (this have to be created)
Anyone can suggest a better name :D